src/uct/ib/efa/ib_efa_dv.c

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include <uct/ib/efa/ib_efa.h>
+
+
+ucs_status_t uct_ib_efadv_query(struct ibv_context *ctx,
+                                uct_ib_efadv_attr_t *efadv_attr)
+{
+    if (efadv_query_device(ctx, efadv_attr, sizeof(*efadv_attr))) {
+        return UCS_ERR_IO_ERROR;
+    }
+    return UCS_OK;
```

Comment:
space line before
print error

Reply:
If we use uct_ib_efadv_query() for EFA device detection too, then I can't print an error here.